scripts/readelf.py

```diff
@@ -1959,12 +1962,19 @@ def main(stream=None):
                 readelf.display_string_dump(args.show_string_dump)
             if args.debug_dump_what:
                 readelf.display_debug_dump(args.debug_dump_what)
+            output.flush()
         except ELFError as ex:
             sys.stdout.flush()
             sys.stderr.write('ELF error: %s\n' % ex)
             if args.show_traceback:
                 traceback.print_exc()
             sys.exit(1)
+        except BrokenPipeError:
```

Comment:
Presumably this follows the guideline at https://docs.python.org/3/library/signal.html#note-on-sigpipe

Why is the exit code here 0 rather than 1?

Reply:
It seemed to me this is not an error but a normal behaviour. That said I have checked readelf from binutils and noticed it returns 141. What value would you like ?